### DISCUSSION TITLE
Unavaliablity of REED (router eligible end devices) in OTNS

### DISCUSSION THREAD

**Discussion by HimanshuTembhe** [openthread/openthread#10313] [Q&A]
why REED option is not avaliable is OTNS although it is mentioned in documentation and FED , MED and SED are included ?

**Comment by jwhui** [comment]
In OTNS, "router" really refers to a "router-capable" device. A router-capable device operates as a router or a REED depending on Thread's router selection process.

**Reply by bimanpaul** [reply]
Can we see REED in action with OTNS like upgrading to Router and Downgrading to End Device?
https://openthread.io/guides/thread-primer/node-roles-and-types#upgrading_and_downgrading

**Reply by jwhui** [reply]
When operating as a child, the node's color should be green.

When operating as a router, the node's color should be blue.

See: https://github.com/openthread/ot-ns/blob/565450bc27399d324d35970e7264f167eb33c19b/web/site/js/vis/Node.js#L224

**Comment by HimanshuTembhe** [comment]
![otns1](https://github.com/openthread/openthread/assets/99601153/3206b44a-f69d-4f4f-8b09-dd34f9d76a63)
I tried various mesh network , but didn't get router downgraded as an end device . In this example also , node 8 should act as a end device but even after 4 hour of simulated time , state of node 8  i.e router is not changed to child .

**Reply by jwhui** [reply]
Per Thread Specification, when there are less than 16 active routers, REEDs will attempt to upgrade to Router.
